### PR TITLE
[improvement](create tablet) be choose disk tolerate with little skew

### PR DESCRIPTION
**Pull request by yujun777**
opt usage skew at 70% and 85%;

## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

File: be/src/olap/storage_engine.cpp [hunk 1]
```diff
@@ -489,11 +491,44 @@ void StorageEngine::_get_candidate_stores(TStorageMedium::type storage_medium,
                 !data_dir->reach_capacity_limit(0)) {
                 DirInfo dir_info;
                 dir_info.data_dir = data_dir;
-                dir_info.available_level = get_available_level(data_dir->get_usage(0));
```
Comment (deardeng) [inline be/src/olap/storage_engine.cpp:492]: get_available_level can remove ?

File: be/src/olap/storage_engine.cpp [hunk 2]
```diff
@@ -489,11 +491,44 @@ void StorageEngine::_get_candidate_stores(TStorageMedium::type storage_medium,
...
+    for (auto usage : usages) {
+        if (usage < 0.7 - (config.high_disk_avail_level_diff_usages / 2.0)) {
+            continue;
+        }
+
+        if (usage > level_max_usages.back() + config::high_disk_avail_level_diff_usages) {
+            level_max_usages.push_back(usage);
+        }
+    }
+    for (auto& dir_info : dir_infos) {
```
Comment (deardeng) [inline be/src/olap/storage_engine.cpp:523]: add some comment?

**Comment by yujun777** [comment]
run buildall

**Review by deardeng — APPROVED** [review]
LGTM

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run p0

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**